frameworks/hdfs/universe/config.json

```diff
@@ -82,25 +82,21 @@
                   "properties": {
                     "hostname": {
                       "type": "string",
-                      "description": "The name or address of a host running a KDC for the realm.",
-                      "default": "kdc.marathon.autoip.dcos.thisdcos.directory"
+                      "description": "The name or address of a host running a KDC for the realm."
```

Comment:
Should these (and the other properties where the defaults have been removed) now be `required`?

Reply:
Short answer: no

Long answer: yes... but only as required if enabled is true. Which the UI does not currently have a good way to display.